src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+@use 'sass:list';
+@import './variables';
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.section {
+  padding: 10px;
+  max-width: calc(($width * 7) + 6px);
```

Comment:
The variable `$width` used here seems to be undefined in this file. Make sure to define it in your `_variables.scss` file or within this file if it's not intended to be a shared variable.